amm/src/main/scala/ammonite/interp/Preprocessor.scala

```diff
@@ -293,12 +309,12 @@ object Preprocessor{
 package ${pkgName.map(_.backticked).mkString(".")}
 ${importBlock(imports)}
 
-object ${indexedWrapperName.backticked}{\n"""
+object ${indexedWrapperName.backticked}{\n""".replace("\n", newLine)
```

Comment:
You should be able to directly interpolate `$newLine` into the string here, and below, rather than leaving the `\n` inside the string and then calling `.replace` to swap it out